web/src/features/AppConfig/components/AppConfig.tsx

```diff
@@ -339,22 +340,6 @@ class AppConfig extends Component<Props, State> {
   };
 
   handleSave = async () => {
-    const [newConfigGroups, hasRequiredFieldError] =
-      this.mergeConfigGroupsAndValidationErrors(
-        this.state.configGroups,
-        [],
-        true
-      );
-
-    if (hasRequiredFieldError) {
-      this.setState({
-        configGroups: newConfigGroups,
-        showErrorsForEmptyValues: true,
-        showValidationError: true,
-      });
-      return;
-    }
-
```

Comment:
this was to run the required field validation on save. we're delegating this to the backend